src/components/accession2/properties/CollectedReceivedDate2.tsx

```diff
@@ -84,7 +86,8 @@ export default function CollectedReceivedDate2({ onChange, record, type, validat
           value={dates.collectedDate}
           onChange={(value) => changeDate('collectedDate', value)}
           errorText={dateErrors.collectedDate}
-          maxDate={Date.now()}
+          maxDate={timeZone ? changeTimezone(new Date(), timeZone) : new Date()}
```

Comment:
leave maxDate as-is, @charlie-tf added a timeZone property to datepicker, use that instead

Reply:
@constanzauanini i made this change it is working for me (i checked out your branch)
keep maxDate={new Date()} and use defaultTimeZone
You have to select the seed-bank 'location dropdown' to see the change.
Select a seedbank that has a time zone you want to test.
If you don't use a seed bank, the default datepicker will show UTC.